forge-gui/res/cardsfolder/upcoming/all_will_be_one.txt

```diff
@@ -0,0 +1,8 @@
+Name:All Will Be One
+ManaCost:3 R R
+Types:Enchantment
+T:Mode$ CounterAddedOnce | TriggerZones$ Battlefield | ValidSource$ You | Execute$ TrigDamage | TriggerDescription$ Whenever you put one or more counters on a permanent or player, CARDNAME deals that much damage to target opponent, creature an opponent controls, or planeswalker an opponent controls.
```

Comment:
The thing is you can't omit the Valid for the Affected because "permanent or player" does not include every possible case (e.g. cards outside battlefield)

Imo you should refactor the trigger into "ValidEntity$" so it can match either

Reply:
I copied something from StaticAbilityIgnoreHexproofShroud which has ValidEntity. Though I'm not sure what card I can use to test if it's working correctly.

Reply:
Hmn, I think I'll try to find time to sneak in a PR for you later

That might honestly be faster, than you can see how it's done

Reply:
Thanks sounds good ! I'll revert my changes